oracle/oracles/pd.go

```diff
@@ -486,8 +485,6 @@ func (o *pdOracle) updateTS(ctx context.Context) {
 	ticker := time.NewTicker(currentInterval)
 	defer ticker.Stop()
 
-	// Note that as `doUpdate` updates last tick time while `nextUpdateInterval` may perform calculation depending on the
-	// last tick time, `doUpdate` should be called after finishing calculating the next interval.
```

Comment:
Looks this is part of #1502 and don't need to be removed

Reply:
Ah yes